share/cmake/modules/Findpybind11.cmake

```diff
@@ -22,28 +23,88 @@ endif()
 ### Try to find package ###
 
 if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
-    set(_pybind11_SEARCH_DIRS
-        ${pybind11_DIRS}
-        ~/Library/Frameworks
-        /Library/Frameworks
-        /usr/local
-        /usr
-        /sw        # Fink
-        /opt/local # DarwinPorts
-        /opt/csw   # Blastwave
-        /opt
-    )
+    if(DEFINED pybind11_DIRS)
+        # Try user defined search path. Ignore all default cmake and system 
+        # paths, and fall back on a system CMake config.
+        find_path(pybind11_INCLUDE_DIR
+            NAMES
+                pybind11/pybind11.h
+            HINTS
+                ${pybind11_DIRS}
+            PATH_SUFFIXES
+                include
+                pybind11/include
+            NO_DEFAULT_PATH
+        )
 
-    # Find include directory
-    find_path(pybind11_INCLUDE_DIR
-        NAMES
-            pybind11/pybind11.h
-        HINTS
-            ${_pybind11_SEARCH_DIRS}
-        PATH_SUFFIXES
-            include
-            pybind11/include
-    )
+        if(EXISTS "${pybind11_INCLUDE_DIR}")
+            # Assume user-provided version is OK.
+            # TODO: Can the pybind11 version be inspected from a header?
```

Comment:
Yes. The code could parse the file `ext/dist/include/pybind11/detail/common.h` to find the version:
```
#define PYBIND11_VERSION_MAJOR 2
#define PYBIND11_VERSION_MINOR 4
#define PYBIND11_VERSION_PATCH 3
```

Reply:
Good find. I should have looked in more of the headers! I will get this incorporated.

Reply:
Added.